torch/csrc/jit/passes/quantization/helper.cpp

```diff
@@ -183,7 +179,15 @@ std::vector<std::string> _tensor_info_funcs = {"size"};
 
 // Aten functions whose output will be quantized or not quantized depending
 // on input tensor
-std::vector<std::string> _propagate_quant_ops = {"cat"};
+std::vector<std::string> _propagate_quant_single_input_ops = {"cat"};
```

Comment:
NIT: Might be better to say "Input list of tensors"

Reply:
cat actually has one input which is a list of tensor, would like to keep the name since this works for all ops that has single input tensor or input list of tensors.